src/const.js

```diff
@@ -1,6 +1,10 @@
 export const DEFLATE_COMPRESSION = 8;
 export const NO_COMPRESSION = 0;
 
+export const SEVERITY_ERROR = 2;
```

Comment:
Yeah, that's better. I used severity as it's what ESLint uses internally, but I didn't like it either. `VALIDATION_` it is! :smile: